angr/engines/vex/light/light.py

```diff
@@ -274,6 +276,16 @@ def _perform_vex_stmt_LLSC(self, result, addr, endness, storedata, ty):
             self._perform_vex_stmt_LLSC_store(addr, storedata, endness)
             self._perform_vex_stmt_LLSC_wrtmp(result, self._handle_vex_const(pyvex.const.U1(1)))
 
+    def _optimize_vex_stmt_guarded_addr(self, guard, addr):
```

Comment:
this can be present as a stub in the light engine, but the implementation should be moved to the claripy mixin (and the actions mixin, for the data layer packing/unpacking)